manager/state/raft/raft.go

```diff
@@ -1142,15 +1145,30 @@ func (n *Node) send(ctx context.Context, messages []raftpb.Message) error {
 			continue
 		}
 
+		ch := make(chan struct{})
```

Comment:
I don't think so, but I might be misunderstanding the suggestion. Can you show some code for what you have in mind?